### DISCUSSION TITLE
Launch Theia, but non appear form: is blank window

### DISCUSSION THREAD

**Discussion by achillepaoloni** [eclipse-theia/theia#13940] [General]
Launch Theia, but non appear form: is blank window

**Comment by msujew** [comment]
Hey @achillepaoloni,

are you talking about the Theia IDE? If so, please create an issue [in this repo](https://github.com/eclipse-theia/theia-blueprint) with reproduction steps.

If you don't, can you please provide some reproduction steps, maybe a combination of OS and Version of Theia?
